distributed/tests/test_worker.py

```diff
@@ -651,3 +652,12 @@ def test_hold_onto_dependents(c, s, a, b):
     yield gen.sleep(0.1)
 
     assert x.key in b.data
+
+
+def test_random_seed(loop):
+    with cluster() as (s, [a, b]):
+        with Client(('127.0.0.1', s['port']), loop=loop) as c:
+            x = c.submit(random.randint, 0, 2**31, pure=False)
+            y = c.submit(random.randint, 0, 2**31, pure=False)
```

Comment:
Should probably be testing for `np.random` as well (since it uses a separate random state).